### ISSUE TITLE
Don't offer custom repositories code completion for official one

### ISSUE THREAD

**Issue by b1rdex** [psliwa/idea-composer-plugin#36]
```
{
    "require": {},
    "require-dev": {
        "phpunit/phpunit": "~5.0"
    },
    "repositories": [
        {
            "type": "composer",
            "url": "https://packagist.org"
        },
        {
            "packagist": false
        }
    ]
}
```

Why this done? To prevent `zlib_decode(): data error`.
Answer is on https://github.com/composer/composer/issues/4892#issuecomment-181943804

**Comment by psliwa** [comment]
Thanks for reporting. The plugin doesn't support completing urls for repositories. What I can do is to add inspection that detects such situations and for example mark such url as error (with the reason of the error). What do you think about this?

**Comment by b1rdex** [comment]
Oh, the issue isn't about url completion. It's about annoying yellow bar with offer to enable code completion (i.e. version names) from custom repositories.
Can't make screenshot because I closed it with 'never' option.

**Comment by psliwa** [comment]
Fixed in `1.0.19`.
